packages/launcher/src/renderer/launcher/identities/IdentityEditModal.js

```diff
@@ -4,15 +4,23 @@ import React, { Component } from 'react'
 import { Text, TextField, Switch } from '@morpheus-ui/core'
 import styled from 'styled-components/native'
 import { type FormSubmitPayload } from '@morpheus-ui/forms'
+import { createFragmentContainer, graphql, commitMutation } from 'react-relay'
 
+import { EnvironmentContext } from '../RelayEnvironment'
 import FormModalView from '../../UIComponents/FormModalView'
-import { type User } from './IdentitiesView'
+import OwnUserIdentity from './__generated__/IdentityEditModal_ownUserIdentity.graphql.js'
```

Comment:
The type is exported as a named export so the import should be something like: `import type { IdentityEditModal_ownUserIdentity as OwnUserIdentity } from './__generated__/IdentityEditModal_ownUserIdentity.graphql'`